### PR TITLE
added celery 4 recipes

### PR DESCRIPTION
**Pull request by Korijn**

### CODE REVIEW

File: kombu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - anyjson >=0.3.3
+        - amqp >=1.4.9,<2.0
```
Comment (Korijn) [inline kombu/meta.yaml]: right here:
https://github.com/celery/kombu/blob/master/requirements/default.txt

Comment (Korijn) [inline kombu/meta.yaml]: oh, kombu no longer depends on anyjson. will remove

File: kombu/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - anyjson >=0.3.3
```
Comment (Korijn) [inline kombu/meta.yaml]: no, they are not. will remove

File: celery/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        - celery = celery.__main__:main
+        - celeryd = celery.__main__:_compat_worker
+        - celerybeat = celery.__main__:_compat_beat
+        - celeryd-multi = celery.__main__:_compat_multi
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - pytz
```
Comment (Korijn) [inline celery/meta.yaml]: nope, will remove

File: vine/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    name: vine
+    version: {{ version }}
+
+source:
+    fn: vine-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/35/21/308904b027636f13c3970ed7caf2c53fca77fa160122ae3ac392d9eb6307/vine-1.1.3.tar.gz
+    md5: 7e56d432c7ba15afcd23416a0ab7725b
+
+build:
+    number: 1
```
Comment (Maxyme) [inline vine/meta.yaml]: build number changed, but not the recipe content.

Comment (Maxyme) [inline vine/meta.yaml]: I recommend having the first recipe at 0.

File: kombu/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    number: 1
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - amqp >=1.4.9,<2.0
```
Comment (Maxyme) [inline kombu/meta.yaml]: I think we should not pin the upper version for packages, unless we know for sure it won't work with later versions, in this case a known incompatibility with 2.0, which we should include in a comment. So in this case I recommend not pinning upper version.

File: celery/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        - celeryd-multi = celery.__main__:_compat_multi
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - pytz
+        - billiard >=3.5.0.2,<3.6.0
```
Comment (Maxyme) [inline celery/meta.yaml]: Pinning upper here. See comment below.

File: vine/meta.yaml (hunk 4 above)
Comment (Maxyme) [inline vine/meta.yaml]: Also valid for other recipes. Since they haven't been part of anything, I recommend they all start at 0.

**Pull request merged**